### ISSUE TITLE
Image Gallery Demo

### ISSUE THREAD

**Issue by zoltan-dulac** [PublicisSapient/enable-a11y#193]
Image Gallery component demo.  Image Galleries need to be coded a particular way to work properly. 

Solution: Make sure the following requirements are in place:
- Side arrows do not get keyboard focus.
- Pagination dots below the carousel get keyboard focus. They must be coded as buttons. Their labels should be "Display photo 1 of X"
- Activating the button with the keyboard moves the panel into view THE SAME WAY AS CLICKING THE MOUSE ON AN ARROW BUTTON.
- Each image in the carousel must have a descriptive alt attribute. When the image moves into view, at the same time, inject the alt attribute content into an empty div with an aria-live="polite" attribute. Since the content within the div is changing, the screen reader will announce the alt attribute.
